### ISSUE TITLE
Feature: Provide slots for terminal components

### ISSUE THREAD

**Issue by TAvsec** [ndabAP/vue-command#101]
It would be great if developers could pass HTML elements to _title_ and _prompt_ fields. In the current version, fields will not get parsed.

![Screenshot](https://user-images.githubusercontent.com/31708774/70429893-dc715000-1a79-11ea-93e2-2fbd22ac59d7.png)

**Comment by ndabAP** [comment]
Thanks for the request.

Interesting idea. I'd rather use `slot` then, so programmers can even introduce components. But it's a heavy API change that needs to be discussed.

**Comment by TAvsec** [comment]
> Thanks for the request.
> 
> Interesting idea. I'd rather use `slot` then, so programmers can even introduce components. But it's a heavy API change that needs to be discussed.
> 
> Do you have an usage example?

Yes, I wanted to change the color of the _prompt_ text, so it would look like in the screenshot below. 
![colorInPrompt](https://user-images.githubusercontent.com/31708774/70454513-d182e380-1aaa-11ea-9b26-213c5459c255.png)

**Comment by krmax44** [comment]
I'll start working on this one once #82 is done & merged.

**Comment by krmax44** [comment]
What about using named slots like this:

```vue
<vue-command>
  <template v-slot:prompt>
    <span>my cool prompt $</span>
  </template>
</vue-command>
```

**Comment by ndabAP** [comment]
Two slots are now supported. For concrete slot wishes please open a new issue.
